### PR TITLE
[zh] translate the observability primer

### PR DESCRIPTION
**Pull request by zhihali**

### CODE REVIEW

**Comment by zhihali** [comment]
May I ask how to fix the i18n CI?

**Comment by svrnm** [comment]
thank you @zhihali 

@open-telemetry/docs-zh-approvers PTAL!

File: content/zh/docs/concepts/observability-primer.md
Comment (my-git9) [inline content/zh/docs/concepts/observability-primer.md:18]:
```suggestion
[追踪](/docs/concepts/signals/traces/)、
[指标](/docs/concepts/signals/metrics/)和
[日志](/docs/concepts/signals/logs/)。当开发人员不需要添加更多插桩就能排查问题时，
```

File: content/zh/docs/concepts/observability-primer.md
Comment (my-git9) [inline content/zh/docs/concepts/observability-primer.md:27]:
```suggestion
**可靠性** 回答了这个问题："服务是否在按用户期望的方式运行？"一个系统可能 100% 的时间都在运行，但如果当用户点击"加入购物车"来添加一双黑色鞋子时，系统并不总是能准确地添加黑色鞋子，那么这个系统就可能是**不**可靠的。
```

File: content/zh/docs/concepts/observability-primer.md
Comment (my-git9) [inline content/zh/docs/concepts/observability-primer.md:45]:
```suggestion
日志样例：
```

File: content/zh/docs/concepts/observability-primer.md
Comment (my-git9) [inline content/zh/docs/concepts/observability-primer.md:53]:
```suggestion
当日志作为 [span](#spans)（跨度）的一部分，或者与 trace（追踪）和 span 关联起来时，它们的价值就会大大增加。
```

**Comment by zhihali** [comment]
@my-git9 Hi, 老哥，我想问一下这种英文或者括号前面加空格的格式规范，在哪里可以系统的看一看呢？约定俗成吗，还是说我们有参考呢？

File: content/zh/docs/concepts/observability-primer.md
Comment (zhihali) [reply content/zh/docs/concepts/observability-primer.md:18]: 太细了，感谢review！

**Comment by my-git9** [comment]
> @my-git9 Hi, 老哥，我想问一下这种英文或者括号前面加空格的格式规范，在哪里可以系统的看一看呢？约定俗成吗，还是说我们有参考呢？

中英文之间

File: content/zh/docs/_index.md
Comment (svrnm) [inline content/zh/docs/_index.md:7]: is there a reason why the anchor has been removed? Is it possible to use an anchor with Chinese symbols? If not you can annotate the title with an anchor name:

```
## Title of the section {#what-is-observability}
```

**Comment by my-git9** [comment]
> @my-git9 为什么这个PR没有merge就close了？是有其他PR冲突了吗？

不好意思，电脑太卡，点错地方了

File: content/zh/docs/_index.md
Comment (chalin) [reply content/zh/docs/_index.md:7]: Thanks for catching that @svrnm. Right, anchors _must not be removed_. URLs should match what is in the English pages. 

@open-telemetry/docs-zh-approvers - please ensure that you follow the best practices outlined in the localization documentation, in this case in particular, see https://opentelemetry.io/docs/contributing/localization/#heading-anchors.

File: content/zh/docs/concepts/observability-primer.md
Comment (chalin) [inline content/zh/docs/concepts/observability-primer.md]: Here and elsewhere, you should preserve the original English links, otherwise the fallback page feature might break.

**Review by chalin — CHANGES_REQUESTED** [review]

File: content/zh/docs/_index.md
Comment (zhihali) [reply content/zh/docs/_index.md]: Thanks for pointing this out, I have updated line9 to include the ` {#what-is-observability}`, could you have a look if it's good now?

File: content/zh/docs/concepts/observability-primer.md
Comment (zhihali) [reply content/zh/docs/concepts/observability-primer.md]: Thanks, I have mark it back. But sometimes I edit the file in my local env, I click the link it will jump to an error place, then I thought this might be a problem, but in fact it could works on the website. So how can I check if every link works well from my local env?

Comment (zhihali) [reply content/zh/docs/concepts/observability-primer.md]: And as you see, if I put this to original English link, the CI check will fail

File: content/zh/docs/concepts/observability-primer.md
Comment (windsonsea) [inline content/zh/docs/concepts/observability-primer.md:14]:
```suggestion
要对你的系统提出这些问题，你的应用程序必须进行适当的插桩。也就是说，应用程序代码必须能够发出[信号](/docs/concepts/signals/)，
```

File: content/zh/docs/concepts/observability-primer.md
Comment (windsonsea) [inline content/zh/docs/concepts/observability-primer.md:16]:
```suggestion
例如[链路](/docs/concepts/signals/traces/)、
```
trace is a noun here, how about to use 链路 instead of 追踪? Seems you can replace it from top to bottom.

File: content/zh/docs/concepts/observability-primer.md
Comment (windsonsea) [inline content/zh/docs/concepts/observability-primer.md:96]:
```suggestion
![链路示例](/img/waterfall-trace.svg '链路瀑布图')
```

**Review by windsonsea — COMMENTED** [review]
Suggest to use 链路 for trace or traces.

File: content/zh/docs/_index.md
Comment (chalin) [reply content/zh/docs/_index.md]: Yes, looks good now, thank you.

**Review by chalin — APPROVED** [review]
LGTM, pending @open-telemetry/docs-zh-approvers approval.

File: content/zh/docs/concepts/observability-primer.md
Comment (zhihali) [reply content/zh/docs/concepts/observability-primer.md:16]: 好的，感谢review

**Review by windsonsea — APPROVED** [review]

**Comment by svrnm** [comment]
It looks like there is another hash that needs to be set:

```
  hash does not exist --- zh/docs/concepts/observability-primer/index.html --> #distributed-traces
```

@zhihali please take a look

**Comment by zhihali** [comment]
> It looks like there is another hash that needs to be set:
> 
> ```
>   hash does not exist --- zh/docs/concepts/observability-primer/index.html --> #distributed-traces
> ```
> 
> @zhihali please take a look

Thanks! @svrnm , have fixed it!

**Comment by svrnm** [comment]
thank you @zhihali

**Pull request merged**
